pkg/resources/function.go

```diff
@@ -179,11 +200,38 @@ func CreateFunction(d *schema.ResourceData, meta interface{}) error {
 		builder.WithLanguage(v.(string))
 	}
 
+	// Set optionals, runtime version for python
+	if v, ok := d.GetOk("runtime_version"); ok {
+		builder.WithRuntimeVersion(v.(string))
+	}
+
+	// Set optionals, warehouse in which to create the function
```

Comment:
so it looks like warehouse is getting set even for non-Python functions. Is this okay? Also i could not find in the documentation where this was supported. Can you pleaselink?